cypress/plugins/main.js

```diff
@@ -104,21 +105,57 @@ function onFunction(on, config) {
           .then(() => firestoreUserToken)
           .finally(() => currentApp.delete());
     },
+
     /**
-     * Produces an EarthEngine token that can be used by production code.
+     * Produces an EarthEngine token that can be used by production code. We use
+     * the somewhat legacy and very poorly documented but still supported
```

Comment:
Can we document anything we know about deprecation timeline and why we're not using the new stuff?

Reply:
Expanded comment a bit. No plans to deprecate, as I understand.